spec/features/visitor_can_browse_items_by_categories_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe "VisitorCanBrowseItemsByCategories", type: :feature do
+  feature "visitor" do
+    let!(:category) { Category.create(name: 'Lenses') }
+    let!(:item_one) { Item.create(name: '70-200 Canon L', description: "portrait lens", price: 12, category_id: category.id) }
```

Comment:
Line is too long. [126/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.